### PR TITLE
Overdose-Digital/magento2-optimizer https://www.wrike.com/open.htm?id…

### PR DESCRIPTION
**Pull request by Mykola-Syniavskyi**
…=679469580

fix issue with error if database config is empty
Added additional checking in AbstractObserver.php, and set default config in config.xml changed version module and CHANGELOG.md

### CODE REVIEW

**Comment by Mykola-Syniavskyi** [comment]
Hello @orthener pls check and merge, Kind regards, Mykola

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 07-06-2021
+### Added
+- additional checking for existing config data in AbstractObserver.php
```
Comment (orthener) [inline CHANGELOG.md]: 1) Better put this line in "Fixed".
2) Use ` or " to wrap file name.
3) Start from capital leter

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
...
+- Set default config fir excluded controllers and paths in config.xml
```
Comment (orthener) [inline CHANGELOG.md]: 1) Put this line in "Changed" section
2) "fir"
3) Use ` or " to wrap file name.

File: Observer/Frontend/Http/AbstractObserver.php [hunk 3]
```diff
@@ -48,7 +48,10 @@ public function __construct(
      */
     public function checkControllersIfExcluded($request, $field, $scope)
     {
-        $excluded_controllers = $this->serializer->unserialize($this->dataHelper->getConfig($field, $scope));
+        $is_set = $this->dataHelper->getConfig($field, $scope);
```
Comment (orthener) [inline Observer/Frontend/Http/AbstractObserver.php]: Bad naming. Use `$isSet`. Same for `$excluded_controllers`, `$current_path`, `$exclude_paths`

File: etc/config.xml [hunk 4]
```diff
@@ -4,10 +4,14 @@
 		<od_optimizer>
 			<move_js_bottom_page>
 				<enable>1</enable>
+                <exclude_controllers>[]</exclude_controllers>
```
Comment (orthener) [inline etc/config.xml:7]: Fix "spaces"

**Pull request merged**